.github/workflows/run_rc_validation.yml

```diff
@@ -93,11 +99,12 @@ jobs:
           touch empty_file.txt
           git add empty_file.txt
           git commit -m "Add empty file in order to create PR" --quiet
-          git push origin ${{env.WORKING_BRANCH}} --quiet
+          git push --force origin ${{env.WORKING_BRANCH}} -f --quiet
```

Comment:
Do we need the `force` flag? If so, could you leave only one please.

Reply:
In my test there were conflicts while trying to push to the destiny branch, but it was probably my branch state, I'm removing it.